src/yaourt.sh.in

```diff
@@ -50,6 +50,7 @@ usage() {
 	echo -e "\t$(gettext 'yaourt {-C --clean}       [options]')"
 	echo -e "\t$(gettext 'yaourt {-B --backup}      [save directory|restore file]')"
 	echo -e "\t$(gettext 'yaourt {-G --getpkgbuild} [options] <package(s)>')"
+	echo -e "\t$(gettext 'yaourt {-P --pkgbuild} [-i --install] <directory>')"
```

Comment:
I think the text in square brackets should be aligned to the similar text above, i.e. moved right by 3 chars.